internal/subscription/core.go

```diff
@@ -22,6 +22,7 @@ type ICore interface {
 	ListKeys(ctx context.Context, prefix string) ([]string, error)
 	List(ctx context.Context, prefix string) ([]*Model, error)
 	Get(ctx context.Context, key string) (*Model, error)
+	Migrate(ctx context.Context, names []string) error
```

Comment:
what about updates happening later, lets say we add more retry durations ? does update api handles updating the topics ?

Reply:
That should go via the normal update subscription route. Already tracking issue here https://github.com/razorpay/metro/issues/234

Reply:
Isn't the function name Migrate a bit generic? We might want to do another kind of migration in future to a different subscription model. In that case will we write another function or are we planning to change the current function's implementation?

Reply:
Sure. I thought of this as throw-away code while writing this to solve the problem at hand. We should be able to modify it as needed in the future as more such use-cases creep in.